manifests/i/IVPN/IVPN/3.4.4/IVPN.IVPN.locale.en-US.yaml

```
@@ -0,0 +1,31 @@
+# Created with YamlCreate.ps1 v2.0.0 $debug=NVS1.5-1-19041-1320
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: IVPN.IVPN
+PackageVersion: 3.4.4
+PackageLocale: en-US
+Publisher: Privatus Limited
```

```suggestion
Publisher: IVPN Limited
```
This actually has to match Control Panel because WinGet uses values straight from the Control Panel.

if you want your publisher to be "Privatus Limited": you'll have to recompile your application with the changes, rather than modifying it in the package.